### ISSUE TITLE
Select chevron not visible in Windows High Contrast Mode (WCAG Level A)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2334]
Page 31 of July 2022 DAC report for EQ/RAS:

-----
Using CSS to convey important information 2
The chevron disappears when colour inversion is implemented.
WCAG Reference:
1.1.1 Non-text Content (Level A)
Understanding Non-text Content | How to Meet Non-text Content
Issue ID: DAC_Using_CSS2
URL: https://eq-v2-prototypes.herokuapp.com/features/long-list-of-codes/long-select
Page title: eQ prototypes
Journey: Task 6 step 1

Screenshot:
<img width="608" alt="Screenshot 2022-07-19 at 14 44 57" src="https://user-images.githubusercontent.com/43346934/179765622-4ae27753-f7ed-4414-98b8-9c749fa2bf6e.png">

The chevron from the select element disappears when colours are inverted due to the black chevron against the black background. This is because it is a background image and is not supported by High Contrast mode.
The chevron provides a visual cue for visual users that the element is a select element and can be actioned to reveal its options.
Windows High Contrast mode can be used by using the following keyboard commands: Shift + alt + Print Screen

Current code ref(s): #select 
```
<select id="select" name="select" class="ons-input ons-input--select"> background: #ffffff url(../img/icons--chevron-down.svg) no-repeat center right 10px;
```

Low vision user comments: “When viewing this page in high contrast, I noticed that the chevron is not displayed in the drop-down box. Because I use Windows high contrast as my default contrast when I am viewing websites this is an issue for me, because I would not know that there is a drop-down box available without the chevron displayed. It would be beneficial if the chevron is clearly displayed, to visibly indicate that it is a dropdown box to access further options.”

Solution:
We recommend referring to the select element example taken from the GOV.UK Design System whereby the select element does not use background images and inverts correctly with a visible chevron when using Windows High Contrast mode.
F3: Failure of Success Criterion 1.1.1 due to using CSS to include images that convey important information
